src/Tests/Integration/fixtures/banks/leumi/08-account-resolve/frame-8.html

```
@@ -0,0 +1,4 @@
+<html>
```

HTMLHint (1.9.2)

[warning] An lang attribute must be present on <html> elements.

(html-lang-require)